File: src/com/java/Day27/AddresBookSystem/UC12.java
package Day27.AddresBookSystem;
import java.util.ArrayList;
import java.util.Comparator;

class ContactUc12 {

    String name;
    String city;

    ContactUc12(String name, String city) {
        this.name = name;
        this.city = city;
    }
}

public class UC12 {

    public static void main(String[] args) {

        ArrayList<ContactUc12> list = new ArrayList<>();

        list.add(new ContactUc12("Ranjith", "Chennai"));
        list.add(new ContactUc12("Arun", "Madurai"));

        list.sort(Comparator.comparing(c -> c.city));

        for (ContactUc12 c : list) {
            System.out.println(c.name + " " + c.city);
        }
    }
}

File: src/Day40/Uc12.js
class EmployeePayrollData {

    get name() {
        return this._name;
    }

    set name(name) {
        this._name = name;
    }

    constructor(name) {
        this.name = name;
    }
}

let employee = new EmployeePayrollData("Ranjith");

console.log(employee.name);